pt_PT.ini

```diff
@@ -183,48 +183,48 @@ VFPU = VFPU
 
 [Dialog]
 Back = Voltar
-Cancel = Cancel
-ChangingGPUBackends = Changing GPU backends requires PPSSPP to restart. Restart now?
-Choose PPSSPP save folder = Choose PPSSPP save folder
-Confirm Overwrite = Do you want to overwrite the data?
-Confirm Save = Do you want to save this data?
-ConfirmLoad = Load this data?
+Cancel = Cancelar
+ChangingGPUBackends = Mudar de motor de GPU requer que o PPSSPP reinicie. Reiniciar agora?
+Choose PPSSPP save folder = Escolher pasta de gravação do PPSSPP
+Confirm Overwrite = Deseja sobreescrever estes dados?
+Confirm Save = Deseja gravar estes dados?
+ConfirmLoad = Carregar estes dados?
 Delete = Apagar
-Delete all = Delete all
-Delete completed = Excluindo com sucesso!
-DeleteConfirm = This save data will be deleted.\nAre you sure you want to continue?
-DeleteConfirmAll = Do you really want to delete all\nyour save data for this game?
-DeleteConfirmGame = Do you really want to delete this game\nfrom your device? You can't undo this.
-DeleteConfirmGameConfig = Do you really want to delete the settings for this game?
-DeleteFailed = Unable to delete data.
+Delete all = Apagar tudo
+Delete completed = Apagado com sucesso!
+DeleteConfirm = Estes dados de gravação serão apagados. Tem a certeza que quer continuar?
+DeleteConfirmAll = Tem a certeza que deseja apagar\ntodos os seus dados gravados deste jogo?
+DeleteConfirmGame = Tem a certeza que deseja apagar\neste jogo do dispositivo? Isto não poderá ser desfeito.
```

Comment:
`Não poderá reverter esta acção.` sounds better than `Isto não poderá ser desfeito.` It's more explicit and draws more the user attention to the implication of his/her action.